src/coreComponents/physicsSolvers/multiphysics/SinglePhasePoromechanicsEmbeddedFractures.cpp

```diff
@@ -601,8 +601,7 @@ void SinglePhasePoromechanicsEmbeddedFractures::updateState( DomainPartition & d
 
       // update the stencil weights using the updated hydraulic aperture
       flowSolver()->updateStencilWeights( domain );
-
-      // update fracture's permeability and porosity
+      // update fracture's porosity from pressure and temperature
```

Comment:
Did you mean "from pressure and aperture?"

Reply:
if you look at what that function does it only updates things based on pressure and temperature.